sdk/src/wallet/account/operations/transaction/sign_transaction.rs

```diff
@@ -24,7 +24,7 @@ impl<S: 'static + SecretManage> Account<S>
 where
     crate::wallet::Error: From<S::Error>,
 {
-    /// Function to sign a transaction essence
+    /// Account method to sign a transaction essence.
```

Comment:
I personally wouldn't put the type, it's a bit obvious and unusual. I'd just sat `Method to...`

Reply:
Fine with me!

Reply:
I changed it to `Signs ...` as we sometimes we do this anyway. So now it should be consistent.